client/src/components/MeetingExperience.tsx

```diff
@@ -89,49 +88,47 @@ export const MeetingExperience = (props: MeetingExperienceProps): JSX.Element =>
     const locale = COMPOSITE_LOCALE_EN_US;
     const formFactorValue = new MobileDetect(window.navigator.userAgent).mobile() ? 'mobile' : 'desktop';
 
-    if (renderPostCall && postCall) {
-      return (
-        <Survey
-          postCall={postCall}
-          onRejoinCall={() => {
-            setRenderPostCall(false);
-            callWithChatAdapter.onStateChange((state: CallWithChatAdapterState) => {
-              //page is set to lobby to avoid flicker of Devices page when "rejoin call" is clicked
-              if (state.page === 'configuration') state.page = 'lobby';
-            });
-            callWithChatAdapter.joinCall();
-          }}
-        />
-      );
-    }
-
     return (
-      <CallWithChatComposite
-        adapter={callWithChatAdapter}
-        fluentTheme={fluentTheme}
-        options={{
-          callControls: {
-            chatButton: chatEnabled
-          }
-        }}
-        locale={{
-          component: locale.component,
-          strings: {
-            chat: locale.strings.chat,
-            call: {
-              ...locale.strings.call,
-              lobbyScreenWaitingToBeAdmittedTitle: waitingTitle,
-              lobbyScreenWaitingToBeAdmittedMoreDetails: waitingSubtitle
-            },
-            callWithChat: locale.strings.callWithChat
-          }
-        }}
-        icons={{
-          LobbyScreenWaitingToBeAdmitted: logo,
-          LobbyScreenConnectingToCall: logo
-        }}
-        formFactor={formFactorValue}
-      />
+      <>
+        {renderPostCall && postCall && (
```

Comment:
We should expand the test to cover the case where `postCall` is falsy and assert that `Survey` is not rendered.

Reply:
'should render CallWithChatComposite when renderPostCall is false' handles this.. renderPostCall being false by default, will get set to true only if postCall has some value for "type" . The reason this "&& postCall' check was added was since postCall was optional, but was being passed as prop to Survey component.

Reply:
I am not sure about that. We don't have a test if postCall is undefined:
```
      <MeetingExperience
        userId={{ communicationUserId: 'test' }}
        token={'token'}
        displayName={'name'}
        endpointUrl={'endpoint'}
        locator={{ meetingLink: 'meeting link' }}
        fluentTheme={undefined}
        waitingTitle={waitingTitle}
        waitingSubtitle={waitingSubtitle}
        logoUrl={logoUrl}
        chatEnabled={true}
        postCall={undefined} <----
        onDisplayError={jest.fn()}
      />
```

Reply:
Yes I see what you mean that it is protected by this condition `if (postCall?.survey.type)`, but we still need to add a unit test for it.

Reply:
done